tests/test_testing.py

```diff
@@ -19,16 +19,23 @@ def test_local_path(nc_file):
 
 @pytest.mark.online
 @pytest.mark.parametrize(
-    ("nc_file", "url_type"), [(nc_file, "opendap"), (nc_file, "http"),]
+    ("nc_file", "url_type", "sub_dir"),
+    [
+        (nc_file, "opendap", "daccs"),
+        (nc_file, "http", "daccs"),
+        (nc_file, "opendap", "climate_explorer_data_prep"),
+    ],
 )
-def test_opendap_path(nc_file, url_type):
+def test_opendap_path(nc_file, url_type, sub_dir):
```

Comment:
I think I forgot to change the name of the test to `test_url_path`. Could you change it in this PR?

Reply:
Oh ya! I noticed that and forgot to change it too.